builtin/logical/database/backend_test.go

```diff
@@ -609,6 +609,40 @@ func TestBackend_allowedRoles(t *testing.T) {
 		t.Fatalf("expected error to be:%s got:%#v\n", logical.ErrPermissionDenied, err)
 	}
 
+	// update connection with glob allowed roles connection
```

Comment:
Can we add a test that has a role name containing a `/`, with a corresponding `allowed_role` having a glob after the `/`?

Reply:
I tested with a `/*` in the strutil package, but role names actually are not able to have `/`s in them. This is because they use the name regex in the path.